R/pin-read-write.R

```diff
@@ -67,6 +69,7 @@ pin_write <- function(board, x,
                       type = NULL,
                       title = NULL,
                       description = NULL,
+                      tags = NULL,
```

Comment:
I wonder if it might be helpful to move the tags argument to the back, to keep backwards compatibility? I'm not sure how often metadata and versioned get used, but the word "tags" is short enough that it should be quick to type!

Reply:
Yeah, after some time, I think this is the right call. 👍